cmd/cosign/cli/options/certificate.go

```diff
@@ -41,14 +40,14 @@ func (o *CertVerifyOptions) AddFlags(cmd *cobra.Command) {
 		"path to the public certificate. The certificate will be verified against the Fulcio roots if the --certificate-chain option is not passed.")
 	_ = cmd.Flags().SetAnnotation("certificate", cobra.BashCompFilenameExt, []string{"cert"})
 
-	cmd.Flags().StringVar(&o.CertEmail, "certificate-email", "",
-		"the email expected in a valid Fulcio certificate")
+	cmd.Flags().StringVar(&o.CertIdentity, "certificate-email", "",
+		"DEPRECATED: Use --certificate-identity instead. The email expected in a valid Fulcio certificate")
 
 	cmd.Flags().StringVar(&o.CertIdentity, "certificate-identity", "",
-		"the identity expected in a valid Fulcio certificate. Valid values include email address, DNS names, IP addresses, and URIs.")
+		"Required. The identity expected in a valid Fulcio certificate. Valid values include email address, DNS names, IP addresses, and URIs.")
```

Comment:
This is a tiny thing, but is it possible to make these options mutually exclusive? Passing both of them at once implies user error/confusion, since only one takes effect at a time.

Reply:
Outdated comment, we're just axeing `--certificate-email` altogether